.travis.yml

```diff
@@ -18,6 +18,7 @@ before_install:
 install:
   - rm -rf ./src/
   - git clone https://github.com/danielperna84/hass-configurator.git ./src
+  - sed -i 's/GIT = False/GIT = True/g' ./src/configurator.py
```

Comment:
I don't think we should create an configuration.py cause it must be overwritten by the user

Reply:
This is modifying the configurator.py script not creating a config file.
git has to be enabled this way as it isnt exposed in the config file

Reply:
alright nevermind got it now ^^